index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="styles.css">
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.0/css/fontawesome.min.css" integrity="sha384-z4tVnCr80ZcL0iufVdGQSUzNvJsKjEtqYZjiQrrYKlpGow+btDHDfQWkFjoaz/Zr" crossorigin="anonymous">
+</head>
+<body>
+    <section class="hero-section">
+        <div class="headline-img"></div>
+        <div class="secondary-cta">
+            <ul class="font-face second-cta container">
+                <li class="s-list"><a class="social" href="#"><img src="./images/twitter.svg" alt="twitter"></a></li>
+                <li class="s-list"><a class="social" href="#"><img src="./images/facebook.svg" alt="facebook"></a></li>
+                <li class="s-list"><a href="#">English</a></li>
+                <li class="s-list"><a href="#">My Page</a></li>
+                <li class="s-list"><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <header>
+            <nav class="container">
+                <div class="logoDiv">
+                    <a class="logo font-face" href="index.html">&lt;/&gt;ITalk</a>
+                </div>
+
+                <img class="mobile-menu pb-4" id="mobileMenu" src="images/ic_menu.svg" alt="Open Navigation">
+                
+                <div class="nav-list">
+                    <span id="mobile-exit">&times;</span>
+                    <ul>
+                        <li class="p-cta"><a href="about.html">About</a></li>
+                        <li class="p-cta"><a href="#">Program</a></li>
+                        <li class="p-cta"><a href="#">join</a></li>
+                        <li class="p-cta"><a href="#">Sponsor</a></li>
+                        <li class="p-cta"><a href="#">News</a></li>
+                        <li class="p-cta call2action"><a href="#">Start Now</a></li>
+                    </ul>
+                </div>
+            </nav>
```

Comment:
- [ ] On mobile screen size, your navbar is expected to be replaced with hamburger on both pages. But at this moment, your navbar still appears on the mobile view port. Kindly look into that and fix it.

### Your design

![your-navbar-issue](https://user-images.githubusercontent.com/26597432/197184487-7d9ade88-c6e0-4d7f-8c7e-88d8f2aa87b3.png)


### Guiding Template

![temp-navbar](https://user-images.githubusercontent.com/26597432/197184546-5d3d7c78-6b49-4bbf-8f6e-8da80c2bcc50.png)

Reply:
@levy002 I am not sure what screen size you are looking at but on the mobile screen, the menu hides and you only see a hamburger menu just like on the design